npm/janus.d.ts

```diff
@@ -74,29 +80,34 @@ declare namespace JanusJS {
 			id?: string;
 			uplink?: number;
 		};
-		error?: Error;
+		error?: string;
+
+		[key: string]: any;
 	}
 
-	interface PluginOptions {
-		plugin: string;
-		opaqueId?: string;
+	interface PluginCallbacks {
 		dataChannelOptions?: RTCDataChannelInit;
 		success?: (handle: PluginHandle) => void;
 		error?: (error: any) => void;
 		consentDialog?: (on: boolean) => void;
 		webrtcState?: (isConnected: boolean) => void;
 		iceState?: (state: 'connected' | 'failed' | 'disconnected' | 'closed') => void;
 		mediaState?: (medium: 'audio' | 'video', receiving: boolean, mid?: number) => void;
-		slowLink?: (uplink: boolean, lost: number) => void;
+		slowLink?: (uplink: boolean, lost: number, mid: string) => void;
 		onmessage?: (message: Message, jsep?: JSEP) => void;
-		onlocalstream?: (stream: MediaStream) => void;
-		onremotestream?: (stream: MediaStream) => void;
+		onlocaltrack?: (track: MediaStreamTrack, on: boolean) => void;
+		onremotetrack?: (track: MediaStreamTrack, mid: string, on: boolean) => void;
 		ondataopen?: Function;
 		ondata?: Function;
 		oncleanup?: Function;
 		detached?: Function;
```

Comment:
this should be `ondetached`

Reply:
This was already there, though, which means it will need to be fixed in the `0.x` branch too.